lib/utils/timezones.ts

```diff
@@ -1201,5 +1201,5 @@ export const getTimeByTimezone = (offset: number) => {
   const time = new Date(utc + 3600000 * offset);
 
   // The formatted time in the specified timezone
-  return format(time, "hh:mmaaa");
+  return format(time, "hh:mm a");
```

Comment:
Could you provide me with more context regarding the change we made? I'm wondering what were the reasons behind it.

Reply:
I'm on my phone right now, but the change basically matches the time formatting with the one that the design had (it was attached to the related issue).